mesa_expense_autopopulate/__manifest__.py

```diff
@@ -0,0 +1,32 @@
+# noinspection PyStatementEffect
+{
+    "name": "Mesa Expense Autopopulate",
+    "category": "Expense",
+    "version": "12.0.0",
+    "author": "Blue Stingray",
+    "website": "https://bluestingray.com/services/erp/",
+    "application": False,
+    "available_in_store": False,
+    "summary": "Autopopulate Expense Forms",
+    "description": "Autopopulate Expense Forms",
+    "depends": ["hr_expense", "sale_expense", "web"],
```

Comment:
```suggestion
    "depends": ["hr_expense", "sale_expense", "web", "gross_margin"],
```

`gross_margin` has the field definitions for `hr.expense` fields: `team_id`, `warehouse_id`, and `user_id`.